### PR TITLE
Extend documentation

### LINKED ISSUE #8: Improve documentation on configuration options
Please add details describing each of the options supported in the `rebar.config`.
In other words, extend [this section](https://github.com/vkatsuba/rebar3_sheldon#config) with explanations about possible values and their expected outcome for each parameter.
Also, if the plugin supports command line options, please add them to the README.

### PR DESCRIPTION
**Pull request by vkatsuba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,7 @@
 [![Build Status][gh badge]][gh]
 [![Erlang Versions][erlang version badge]][gh]
 
-
-A rebar plugin for spellchecking.
+A rebar plugin for spellchecking which is built based on [sheldon](https://github.com/inaka/sheldon).
```
Comment (elbrujohalcon) [inline README.md]:
```suggestion
A rebar plugin for spellchecking code with [Sheldon](https://github.com/inaka/sheldon).
```

File: README.md [hunk 2]
```diff
@@ -28,21 +27,70 @@ $ rebar3 spellcheck
...
+```
+### Short full example
+```sh
+$ rebar3 spellcheck -f 'src/*.erl, test/*erl' -i 'include/*.hrl' -r '[_@./#&+-=*]' -d 'path/to/dict.txt' -a 'path/to.txt, additional_dict_1.txt'`
+```
+
 ## Config
-Example:
+### Description
+By default is used dictionary of [sheldon](https://github.com/inaka/sheldon).
```
Comment (elbrujohalcon) [inline README.md]:
```suggestion
By default, the dictionary used is the one provided by [sheldon](https://github.com/inaka/sheldon).
```

File: README.md [hunk 3]
```diff
@@ -28,21 +27,70 @@ $ rebar3 spellcheck
...
+If need use custom list of files for spellchecking - use config option `files`.
```
Comment (elbrujohalcon) [inline README.md]:
```suggestion
If you need to use a custom list of files for spellchecking, use the config option `files`.
```

File: README.md [hunk 4]
```diff
@@ -28,21 +27,70 @@ $ rebar3 spellcheck
...
+If need ignore some files for spellchecking - use config option `ignore`.
```
Comment (elbrujohalcon) [inline README.md]:
```suggestion
If you need to ignore some files for spellchecking, use the config option `ignore`.
```

File: README.md [hunk 5]
```diff
@@ -28,21 +27,70 @@ $ rebar3 spellcheck
...
+If need ignore some lines - use `ignore_regex` for set regular exemptions for ignore lines.
```
Comment (elbrujohalcon) [inline README.md]:
```suggestion
If you need to ignore some particular strings or comments in your modules, use `ignore_regex` to set regular expressions and ignore the lines that match them.
```

File: README.md [hunk 6]
```diff
@@ -28,21 +27,70 @@ $ rebar3 spellcheck
...
+If `sheldon` dictionary is not suite at all - can be replaced by config option `default_dictionary`.
```
Comment (elbrujohalcon) [inline README.md]:
```suggestion
If you want to directly replace Sheldon's dictionary with your own one, you can replace using the config option `default_dictionary`.
```

File: README.md [hunk 7]
```diff
@@ -28,21 +27,70 @@ $ rebar3 spellcheck
...
+If `sheldon` dictionary is suitable but should be expanded - use `additional_dictionaries` option to set own dictionaries for expand `sheldon` dictionary.
```
Comment (elbrujohalcon) [inline README.md]:
```suggestion
If you need to _expand_ Sheldon's dictionary with your own words, use the `additional_dictionaries` option pointing to your own dictionaries with which you want to expand Sheldon's.
```

File: README.md [hunk 8]
```diff
@@ -28,21 +27,70 @@ $ rebar3 spellcheck
...
+
+### Options
+Currently supported next options of `spellcheck` configuration:
```
Comment (elbrujohalcon) [inline README.md]:
```suggestion
Currently supported options for `spellcheck` configuration through `rebar.config`:
```

**Review by elbrujohalcon — CHANGES_REQUESTED** [review]
Some English tidying up.

**Comment by vkatsuba** [comment]
> Some English tidying up.

Done. Thanks.

**Review by elbrujohalcon — APPROVED** [review]

**Pull request merged**